stories/landsat5.eocloud.stories.js

```diff
@@ -0,0 +1,185 @@
+  const img = document.createElement('img');
+  img.width = '512';
+  img.height = '512';
+
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = "<h2>GetMapUrl (WMS)</h2>";
+  wrapperEl.insertAdjacentElement("beforeend", img);
+
+  const layer = new Landsat5EOCloudLayer(instanceId, layerId);
+
+  const getMapParams = {
+    bbox: bbox,
+    fromTime: new Date(Date.UTC(2010, 11 - 1, 22, 0, 0, 0)),
+    toTime: new Date(Date.UTC(2010, 12 - 1, 22, 23, 59, 59)),
+    width: 512,
+    height: 512,
+    format: MimeTypes.JPEG,
+  };
+  const imageUrl = layer.getMapUrl(getMapParams, ApiType.WMS);
+  img.src = imageUrl;
+
+  return wrapperEl;
+};
+
+export const getMapWMS = () => {
+  const img = document.createElement('img');
+  img.width = '512';
+  img.height = '512';
+
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = "<h2>GetMap with WMS</h2>";
+  wrapperEl.insertAdjacentElement("beforeend", img);
+
+  const perform = async () => {
+    const layer = new Landsat5EOCloudLayer(instanceId, layerId);
+
+    const getMapParams = {
+      bbox: bbox,
+      fromTime: new Date(Date.UTC(2010, 11 - 1, 22, 0, 0, 0)),
+      toTime: new Date(Date.UTC(2010, 12 - 1, 22, 23, 59, 59)),
+      width: 512,
+      height: 512,
+      format: MimeTypes.JPEG,
+    };
+    const imageBlob = await layer.getMap(getMapParams, ApiType.WMS);
+    img.src = URL.createObjectURL(imageBlob);
+  };
+  perform().then(() => {});
+
+  return wrapperEl;
+};
+
+export const getMapWMSLayersFactory = () => {
+  const img = document.createElement('img');
+  img.width = '512';
+  img.height = '512';
+
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = "<h2>GetMap with WMS</h2>";
+  wrapperEl.insertAdjacentElement("beforeend", img);
+
+  const perform = async () => {
+    const layer = (await LayersFactory.makeLayers(`${DATASET_EOCLOUD_LANDSAT5.shServiceHostname}v1/wms/${instanceId}`, (lId, datasetId) => (layerId === lId)))[0];
+
+    const getMapParams = {
+      bbox: bbox,
+      fromTime: new Date(Date.UTC(2010, 11 - 1, 22, 0, 0, 0)),
+      toTime: new Date(Date.UTC(2010, 12 - 1, 22, 23, 59, 59)),
+      width: 512,
+      height: 512,
+      format: MimeTypes.JPEG,
+    };
+    const imageBlob = await layer.getMap(getMapParams, ApiType.WMS);
+    img.src = URL.createObjectURL(imageBlob);
+  };
+  perform().then(() => {});
+
+  return wrapperEl;
+};
+
+export const getMapWMSEvalscript = () => {
+  const img = document.createElement('img');
+  img.width = '512';
+  img.height = '512';
+
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = "<h2>GetMap with WMS - evalscript</h2>";
+  wrapperEl.insertAdjacentElement("beforeend", img);
+
+  const perform = async () => {
+    const layer = new Landsat5EOCloudLayer(
+      instanceId,
+      layerId,
+      `
+        return [2.5 * B04, 1.5 * B03, 0.5 * B02];
+      `,
+    );
+    console.log("aaaaaaaaaaaaaaaaaaa", layer)
```

Comment:
:D
Maybe a bit more descriptive message like `Landsat 5 layer with evalscript` would be better, if it's even needed.

Reply:
:D Thanks, fixed. I have already noticed and fixed it in #20, but I agree, it belongs here, so I have backported the change to this branch.

Reply:
Resolving.